dateparser/date.py

```diff
@@ -14,6 +14,12 @@
 from dateparser.languages.detection import AutoDetectLanguage, ExactLanguages
 
 
+APOSTROPHE_LOOK_ALIKE_CHARS = [
+    u'\u2019', u'\u02bc', u'\u02bb', u'\u055a', u'\ua78c',
+    u'\uff07', u'\u2032', u'\u2035', u'\u02b9', u'\uff07',
```

Comment:
@waqasshabbir  Hmm, isn't it more readable to use the UTF code since those characters are looks alike? I named the array `APOSTROPHE_LOOK_ALIKE_CHARS` so developer reading this will know that those characters are similar to apostrophe (in fact u"\u2019" is the correct apostrophe character). But maybe I am being subjective here, what do you think @asadurski ?